components/raftstore-v2/src/raft/peer.rs

```diff
@@ -624,6 +624,22 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             if let Some(instant) = self.peer_heartbeats.get(&p.get_id()) {
                 let elapsed = now.saturating_duration_since(*instant);
                 if elapsed >= ctx.cfg.max_peer_down_duration.0 {
+                    if !self
+                        .abnormal_peer_context()
+                        .down_peers()
+                        .contains(&p.get_id())
+                    {
+                        warn!(
+                        self.logger,
+                        "peer is down";
```

Comment:
When one store is down, it would print a lot of logs at a time

Reply:
yes.

Reply:
better print the down peers which are not from down store